package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "react-native-bluetooth-state",
-  "version": "2.0.0",
+  "version": "2.1.0",
+  "description": "React Native Module for reading the bluetooth state on iOS and Android (fork of frostney/react-native-bluetooth-state)",
+  "author": "Johannes Stein (http://github.com/frostney)",
+  "contributors": "Christian Ost",
   "repository": {
     "type": "git",
-    "url": "https://github.com/frostney/react-native-bluetooth-state.git"
+    "url": "https://github.com/Artirigo/react-native-bluetooth-state.git"
```

Comment:
Also here: Sure!
Is seen as a temporary change from me to fit the other repository, would absolutely set it back when closing the pull request and before publishing it to npm!